content/en/logs/guide/manage-sensitive-logs-data-access.md

```diff
@@ -40,6 +37,20 @@ If you have already indexed logs that contain sensitive data, then follow these
 2. [Fix the source of the data upstream](#fix-the-source-of-the-data-upstream)
 3. [Handle data already sent to Datadog](#handle-data-already-sent-to-and-indexed-in-datadog)
 
+### Mask action
+
```

Comment:
Once the content for the section is finalized, move to a shortcode. Will also use shortcode in the Manage Sensitive Logs Data Access doc, so that there is single source of truth for the content.

Reply:
done.